docs/mmds/mmds-user-guide.md

```diff
@@ -21,8 +21,8 @@ curl --unix-socket /tmp/firecracker.socket -i                 \
   -X PUT 'http://localhost/network-interfaces/${MMDS_NET_IF}' \
   -H 'Accept: application/json'                               \
   -H 'Content-Type: application/json'                         \
-  -d '{
       "iface_id": "${MMDS_NET_IF}",
+  -d '{
```

Comment:
This is misplaced. It should contain `iface_id` field as well.

Reply:
Oops. Fixed. Bad vim foo there.